docker/README.md

```diff
@@ -207,65 +238,91 @@ And set permissions, so any user can read/write the data:
 
     chmod -R 777 data
 
-This step is unfortunately necessary because Docker [can't mount volume with specific user ownership](https://github.com/docker/docker/issues/7198) in container, so ownership of mounted folders (uid and gid) is the same as on your host system (presumably 1000:1000 which maps into nonexistent user in all of the containers).
+This step is unfortunately necessary because Docker
+[can not mount volume with specific user ownership](https://github.com/docker/docker/issues/7198)
+in container, so ownership of mounted folders (uid and gid) is the same as on
+your host system (presumably 1000:1000 which maps into nonexistent user in all
+of the containers).
 
-If you wish to keep the tests (for example) separate from the shared directory, for any reason (we do, in our development scenario) refer to the [Developing tests with Container setup] section at the end of this document.
+If you wish to keep the tests (for example) separate from the shared
+directory, for any reason (we do, in our development scenario) refer to the
+[Developing tests with Container setup] section at the end of this document.
 
-Populate the OpenQA's database:
+Populate the OpenQA database:
```

Comment:
```suggestion
Populate the openQA database:
```

Reply:
done along with other references showing the same